j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/BytecodeTask.java

```diff
@@ -119,4 +152,23 @@ public Task resolve(Project project, Config config) {
     protected File getGeneratedClassesDir(TaskContext context) {
         return context.outputPath().toFile();
     }
+
+    private Set<String> maybeAddInReactorAptProcessor(List<Input> reactorProcessors, Set<String> processors) {
+        if(processors.isEmpty()) {
+            return Collections.emptySet();
+        }
+        Set<String> existingProcessors = new HashSet<>(processors);
+        reactorProcessors.stream().map(input -> input.filter(APT_PROCESSOR))
```

Comment:
It should actually be an error to call Input.filter() within the task itself - only should be called in the factory before the task is made. But that isn't enforced...